python/python/lancedb/_lancedb.pyi

```diff
@@ -66,6 +68,7 @@ class Query:
     def limit(self, limit: int): ...
     def offset(self, offset: int): ...
     def nearest_to(self, query_vec: pa.Array) -> VectorQuery: ...
+    def rerank(self, Reranker) -> FTSQuery: ...
```

Comment:
I don't believe this change is needed. These definitions are effectively to give us intellisense for the underlying rust types in this file:
https://github.com/lancedb/lancedb/blob/main/python/src/query.rs#L146

But since we haven't modified this Rust code (and I don't think we'll need to for this PR, since our rerankers are already all written in python), we won't need to make changes in this file either.